ui/src/App.tsx

```diff
@@ -1,31 +1,47 @@
 import * as React from "react";
+import { SWRConfig } from "swr";
 import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
 import { Home, ShoeMap } from "./pages";
 
+const defaultFetcher = (
+  input: RequestInfo,
+  init?: RequestInit
+): Promise<unknown> =>
+  fetch(input, init)
+    .then((res) => res.json())
+    .then(({ payload }) => payload);
+
 function App(): JSX.Element {
   return (
-    <Router>
-      <nav>
-        <ul>
-          <li>
-            <Link to="/">Home</Link>
-          </li>
-          <li>
-            <Link to="/map">ShoeMap</Link>
-          </li>
-        </ul>
-      </nav>
+    <SWRConfig
+      value={{
+        fetcher: defaultFetcher,
+        dedupingInterval: 0,
```

Comment:
I am not sure we should remove deduping by setting this to 0, our stories and authors should not change that frequently, so reducing duplicate requests for the sake of efficiency should be fine.

Reply:
Yup, good catch put that in there for testing. I can change.